cumulus/client/consensus/aura/src/collators/slot_based/block_import.rs

```diff
@@ -101,6 +153,15 @@ where
 		// means that the node produced the block itself or the block was imported via state sync.
 		if !self.sender.is_closed() && !matches!(params.state_action, StateAction::ApplyChanges(_))
 		{
+			validate_block_import::<_, _, P, _>(
+				&mut params,
+				self.client.as_ref(),
+				&self.create_inherent_data_providers,
+				self.check_for_equivocation,
+				&self.compatibility_mode,
+			)
+			.await?;
```

Comment:
Given this is a continuation of the verification step, shouldn't this be done no matter what (so the first thing done in the `import_block` method)?